pages/faq/components/FAQsection.tsx

```diff
@@ -48,21 +49,25 @@ export const FAQsection: FC<IFAQsection> = ({
 				</ListItemButton>
 				<Collapse in={open} timeout="auto" unmountOnExit>
 					<List component="div" disablePadding>
-						{faqs.map(({ faq, desc }, i) => (
-							<Box key={`${title}-faq-${i}`}>
-								<Divider />
-								<Accordion sx={{ px: 4, py: 1 }} elevation={0}>
-									<AccordionSummary expandIcon={<ExpandMore color="primary" />}>
-										<Typography variant="subtitle1" color="text.secondary">
-											{faq}
-										</Typography>
-									</AccordionSummary>
-									<AccordionDetails>
-										<Typography variant="body1">{desc}</Typography>
-									</AccordionDetails>
-								</Accordion>
-							</Box>
-						))}
+						<ul className={styles.list}>
```

Comment:
No need to create extra `ul` elemenet.

You can create the `ul` element by editing Line 51
`<List component="div" disablePadding>` ---> `<List component="ul" disablePadding>` (or maybe simply `<List  disablePadding>`)